### PR TITLE
Fix #75, add timeout to docker options based on preview task

### LINKED ISSUE #75: Use ttl from payload in deployer

### PR DESCRIPTION
**Pull request by relekang**

### CODE REVIEW

File: frigg_worker/fetcher.py [hunk 1]
```diff
@@ -53,6 +53,7 @@ def start_build(task, options):
 def start_deployment(task, options):
     docker_options = {
         'image': task['image'],
+        'timeout': task['ttl'],
```
Comment (frecar) [inline frigg_worker/fetcher.py]: this will throw a keyerror if not provided, but I guess that's ok?

Comment (relekang) [inline frigg_worker/fetcher.py]: Yes, but `ttl` should be a part of all preview payloads.

Comment (frecar) [inline frigg_worker/fetcher.py]: yup :)

**Comment by frecar** [comment]
:+1:

**Pull request merged**